Gruntfile.js

```diff
@@ -79,7 +79,7 @@ module.exports = function( grunt ) {
 			},
 			test: {
 				src: [ "test/*.js", "test/functional/**/*.js", "test/unit/**/*.js",
-					"!test/config.js" ],
+					"test/compiler/**/*.js", "!test/config.js", "!test/compiler/_compiled/**" ],
```

Comment:
I think it has to be inside the working directory (so it can't be `os.tmpdir()`), because it needs to find `node_modules`.